cli/chaosli/cmd/context.go

```diff
@@ -0,0 +1,393 @@
+package cmd
+
+import (
+	"encoding/json"
```

Comment:
imports need sorted

Reply:
Can you explain, goland sorts and places the imports for me, unsure how to further sort (also tried to apply gofmt with no change).

Reply:
This does look like the default `gofmt` rather than `goimports`, which appears to be the convention we use looking at `disruption_controller.go` where imports are split into three groups: stdlib, external packages, local packages

Reply:
got it, thanks!